index.html

```diff
@@ -25,15 +25,212 @@
 
         <section class="headline row">
             <div class="container">
-                <h1 class="headline-name darkblue">Binyam Y. Olango</h1>
-                <p class="headline-desc darkblue">Hello! I am a software developer! I can help you build a product, feature or website. 
+                <h1 class="title darkblue">Binyam Y. Olango</h1>
+                <p class="par-desc darkblue">Hello! I am a software developer! I can help you build a product, feature or website. 
                     Take a look at my work.<br><br>If you like what you see and have a project you need coded, 
                     don’t hesitate to contact me.</p>
-                <button class="headline-btn">Start collaboration</button>
+                <button class="btn">Start collaboration</button>
             </div>
 
             <div class="headline-bottom-design"></div>
         </section>
+
+        <section class="work">
+            <div class="container">
+                <h2 class="title darkblue">Projects</h2>
+            </div>
+            <div class="grid">
+                <div class="work1 work-cont">
+                    <figure class="work-fig">
+                        <img src="proj-img/Rectangle 21.svg" alt="work1-image" class="work-img">
+                        <figcaption class="workDesc">
+                            <div class="container">
+                                <p>Project name goes here</p>
```

Comment:
- [ ] please kindly make use of the semantic header`hx` tags such as ` h2, h3, h4, h5` etc to display the title for each card. It is important to display your project names using the semantic heading tag as this will comply with proper html standards
- [ ] please enusre you make this change in every single card in your works section